Documentation/contributor-guide/community-membership.md

```diff
@@ -115,6 +115,14 @@ depend on to make decisions in the best interest of the project in a consistent
 - Subscribed to [etcd-maintainers@googlegroups.com]
 - Elected by supermajority of active maintainers.
 
+### Onboarding process
```

Comment:
Please mention private maintainer Slack.

Reply:
Got it